version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1"
```

Comment:
I thought we should publish the first version as 0.1 directly and then change it to snapshot ?